tests/testthat/test-bind_marginals.R

```diff
@@ -0,0 +1,9 @@
+test_that("bind_marginals works", {
```

Comment:
`rnorm()` does not return a distribution object. Use `distionary::dst_norm()` to create a Normal distribution.